i18n/translations.fi.json

```diff
@@ -120,10 +121,46 @@
         "heading": "Välistys",
         "ingress": "Lorem ipsum asdasd"
     },
+    "patterns": {
+        "general": "Yleistä suunnittelumalleista",
+        "multi_insert": "Multi-insert"
+    },
+    "patterns_main_page": {
+
+    },
+    "multi-insert": {
+        "heading": "Multi-insert",
+        "ingress": "Lomakkeissa on toisinaan tarpeen lisätä sama moduuli eli tietokenttä tai tietokenttien yhdistelmä useaan kertaan. Tämän kaltaisen dynaamisen toiminnallisuuden toteuttamisessa on kiinnitettävä erityistä huomiota käytettävyyteen ja saavutettavuuteen.",
+        "accessibility_list": {
+            "point_1": "Multi-insert-suunnittelumalli vaatii erityistä huomiota fokuksen oikean siirtymisen suhteen. Fokuksen siirtyminen on kuvattu erikseen jäljempänä tällä sivulla.",
+            "point_2": "Jokaisen moduulin yhteydessä olevaan poista-painikkeeseen tulee liittää ruudunlukijaa varten uniikki tieto siitä tiedosta, jota ollaan poistamassa. Referenssitoteutuksessa on esitetty, miten painikkeen aria-label muuttuu dynaamisesti moduulin tärkeimmän tietokentän arvon mukaan.",
+            "point_3": "Moduulin poistamisen jälkeen näytetään ponnahdusilmoitus (toast), jolle on asetettu rooliksi alert. Tämän johdosta myös ruudunlukijakäyttäjät saavat tiedon poistamisesta.",
+            "point_4": "Mikäli lomaketta tallennettaessa moduuleita on jäänyt kokonaan tyhjäksi, ei niitä huomioida, vaan ne poistetaan tallennetusta versiosta. Jos pakollisia tietoja on jäänyt puuttumaan moduulin sisällä, ne indikoidaan kuten muutkin pakolliset tiedot."
+        },
+        "components_text": "Malli rakentuu seuraavista komponenteista:",
+        "focus": {
+            "heading": "Fokuksen siirtäminen",
+            "point_1": "Moduuleita lisättäessä fokus siirtyy juuri lisätyn moduulin ensimmäiseen tietokenttään.",
+            "point_2": "Moduuleita poistettaessa fokus siirtyy \"Lisää uusi\" -painikkeeseen mikäli moduuleita oli vain yksi.",
```

Comment:
Fokuksen siirto moduuleita poistettaessa voisi avautua vielä paremmin, jos olisi alussa tuo tilanne:
Moduulia poistaessa
- mikäli moduuleita on yksi, fokus siirtyy...
- mikäli moduuleita on useita ja viimeinen moduuli poistetaan, fokus siirty... 
- mikäli moduuleita on useita ja muu kuin viimeinen moduuli poistetaan, fokus siirtyy...

Reply:
Check. Muutin tekstiä, onko parempi?

Reply:
Joo mun mielestä helpompi hahmottaa näin, kun aika monimutkaista tuo fokuksen siirtely! Vaihtaisin vielä nuo oli -> on

Reply:
Check